test/test-fs.c

```diff
@@ -851,7 +851,8 @@ static void check_utime(const char* path,
 #endif
     st_atim = s->st_atim.tv_sec + s->st_atim.tv_nsec / 1e9;
     st_mtim = s->st_mtim.tv_sec + s->st_mtim.tv_nsec / 1e9;
-    ASSERT_DOUBLE_EQ(st_atim, atime);
+    if (!test_lutime)
```

Comment:
Can you add a comment explaining what you wrote in the commit log? LGTM apart from that and thanks for fixing it.